client-v2/src/app/components/organisms/task/task.component.html

```diff
@@ -0,0 +1,59 @@
+<li
+    class="task | flex w-full max-w-full list-none flex-col rounded-xl bg-tinted-800 py-2 px-1 pb-1 transition-colors duration-75"
+    [class]="{
+        'bg-submit-800': data.status == TaskStatus.COMPLETED,
+        'text-submit-200': data.status == TaskStatus.COMPLETED,
+        completed: data.status == TaskStatus.COMPLETED,
+        titleHasFocus: titleFocusable.isFocused
```

Comment:
Lets also add `notesHasFocus` here to make other notes-dependent selectors work without `:has()`